internal/domain/pull_request.go

```diff
@@ -0,0 +1,30 @@
+package domain
+
+type PullRequestEvent struct {
+	Action      string      `json:"action"`
+	PullRequest PullRequest `json:"pull_request"`
+	Repository  Repository  `json:"repository"`
+}
+
+type GitHubUser struct {
+	Login string `json:"login"`
```

Comment:
Consider adding validation or constraints on the struct fields to ensure data integrity, especially for fields like `Id`, `Number`, and `Url`. This can help in preventing incorrect data formats from being assigned to these properties.